about.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous">
+    <link rel="stylesheet" href="styles.css">
+    <title>+254 🇰🇪 Concerts: about</title>
+</head>
+
+<body>
+    <section>
+        <div class="container-fluid intro d-flex flex-column col-sm-12 col-md-12">
+            <header>
+                <div class="mobile-navbar">
+                    <div class="container">
+                        <ul class="mobile">
+                            <li class="nav-mobile"><a href="index.html" class="event-page">Home</a></li>
+                            <li class="nav-mobile"><a href="about.html" class="about">Event</a></li>
+                            <li class="nav-mobile"><a href="#logo" class="logo">Logo</a></li>
+                        </ul>
+                        <div class="hamburger">
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                            <span class="bar"></span>
+                        </div>
+                    </div>
+                </div>
+                <div class="desk-navbar fixed-top">
+                <div class="black_nav_desktop">
+                    <div class="nav_logos_desktop">
+                      <a href="www.facebook.com"><img class="fb" src="images/facebook-new.png" alt="facebook logo"></a>
+                      <a href="www.twitter.com"><img class="twitter" src="images/twitter-new.png" alt="twitter logo"></a>
+                    </div>
+                    <a href="#">English</a>
+                    <a href="#">My Page</a>
+                    <a href="#">Korean</a>
+                  </div>
+                  <div class="desk-menu">
+                    <ul class="menu">
+                      <a href="index.html"><img class="logo-desk" src="images/lion-logo.png" alt="logo"></a>
+                      <li><a id="about-desk" href="about.html">About</a></li>
```

Comment:
- [ ] You should also ensure that when I click the About button, I should be able to go back to the Home page. Currently, this feature is only working in the mobile version. Kindly implement it in the Desktop version also ☺️ .

Reply:
Hi @AlphaNtihinduka., thank you for your review., however, I do not understand the above point. How would I implement that? The project description asked that the logo be the one to take you back home. Please advise.